.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📖 OpenFGA's Documentation
+    url: https://openfga.dev/docs
+    about: Check OpenFGA's documentation for an in-depth overview
+  - name: 👽 Discord
+    url: https://discord.gg/8naAwJfWN6
+    about: Join OpenFGA's active Discord community
+  - name: 📝 RFC's
```

Comment:
```suggestion
  - name: 📝 RFCs
```

Reply:
Sorry missed this earlier 🙈

Reply:
All good @rhamzeh ⭐, appreciate the keen eye 👁️

Reply:
Accounted for within [`47b46a0` (#1262)](https://github.com/openfga/openfga/pull/1262/commits/47b46a0b2c747db9857ae2fc9e75b8bcb4364b66)